ingestion-beam/src/test/java/com/mozilla/telemetry/avro/PubsubMessageRecordFormatterTest.java

```diff
@@ -9,17 +9,14 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.Map;
-
 import org.apache.avro.AvroTypeException;
 import org.apache.avro.Schema;
 import org.apache.avro.SchemaBuilder;
 import org.apache.avro.generic.GenericArray;
 import org.apache.avro.generic.GenericRecord;
 import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
-
 import org.json.JSONArray;
 import org.json.JSONObject;
-
```

Comment:
r? @acmiyaguchi because this removes the package separation you used in the avro stuff, bringing it more in line with the rest of the code base